stk/calculators/optimization/optimizers.py

```diff
@@ -773,19 +1178,18 @@ def optimize(self, mol, conformer=-1):
 
         os.mkdir(output_dir)
         init_dir = os.getcwd()
+        os.chdir(output_dir)
+
         try:
-            os.chdir(output_dir)
-            xyz = join(output_dir, f'input_structure.xyz')
-            mol.write(xyz, conformer=conformer)
-            cmd = [
-                self.gfnxtb_path,
-                xyz,
-                '-opt',
-                '--parallel',
-                self.num_cores
-            ]
-            sp.run(cmd, stdin=sp.PIPE, stdout=sp.PIPE, stderr=sp.PIPE)
-            output_xyz = join(output_dir, 'xtbopt.xyz')
-            mol.update_from_xyz(path=output_xyz, conformer=conformer)
+            result = self._run_optimizations(mol, conformer)
```

Comment:
how about changing result to "complete" 
which will always be a True or False now given the _run_optimizeations changes.